tasks/get_psql_version.rb

```diff
@@ -0,0 +1,24 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+# frozen_string_literal: true
+
+require '/opt/puppetlabs/puppet/cache/lib/pe_install/pe_postgresql_info.rb'
+
+# GetPEAdmConfig task class
+class GetPSQLInfo
```

Comment:
This doesn't use the ruby task helper, because it doesn't need param processing?

Reply:
I didn't think that through, just copied the structure of an existing similar task [peadm::get_peadm_config](https://github.com/puppetlabs/puppetlabs-peadm/blob/main/tasks/get_peadm_config.rb)

Reply:
@ody Understood, but in that case - isn't it overkill to create a Ruby class instead of just using 3 lines of procedural code?

Reply:
You are correct. The usage of a Ruby class is a simple side effect of my copy and pasting existing code.

Reply:
The class wrapper is present in the pattern in order to enable unit testing, should unit tests be written.

We should probably just adopt `ruby_task_helper` instead. It's better to standardize on a standard rather than create our own inside PEAdm. If we do that, we also gain the ability to write unit tests for tasks.